app/controllers/sites_controller.rb

```diff
@@ -42,7 +42,7 @@ def show
   def new
     do_new_resource
     get_project_if_exists
-    do_set_attributes
+    do_set_attributes(site_params(create: true))
```

Comment:
```suggestion
    do_set_attributes(site_params(for_create: true))
```

It looks like the parameter should be `for_create:` instead of `create:`